src/main/scala/it/unibo/authsim/library/dsl/cryptography/algorithm/symmetric/DES.scala

```diff
@@ -0,0 +1,48 @@
+package it.unibo.authsim.library.dsl.cryptography.algorithm.symmetric
+
+import it.unibo.authsim.library.dsl.cryptography.algorithm.SymmetricEncryptionAlgorithm
+
+
+trait DES extends SymmetricEncryptionAlgorithm
+
+object DES:
+  import it.unibo.authsim.library.dsl.cryptography.encrypter.asymmetric.key.KeysGenerator
+  import it.unibo.authsim.library.dsl.cryptography.util.ImplicitConversion._
+
+  def apply()= new BasicDES()
+
+  case class BasicDES() extends DES:
+
+    type Salt = String
+
+    private var _length : Int = 7 //bit or 64
+    
+    /**
+     * Private variable representing the algorithm name
+     */
+    val _name : String ="DES"
+
+    /**
+     * Private variable representing the salt value
+     */
+    private var _salt: Array[Byte] = Array(0xA9.asInstanceOf[Byte], 0x9B.asInstanceOf[Byte], 0xC8.asInstanceOf[Byte], 0x32.asInstanceOf[Byte], 0x56.asInstanceOf[Byte], 0x35.asInstanceOf[Byte], 0xE3.asInstanceOf[Byte], 0x03.asInstanceOf[Byte])
```

Comment:
Consider using `toByte` rather than `asInstanceOf[Byte]`:
```suggestion
    private var _salt: Array[Byte] = Array(0xA9.toByte, 0x9B.toByte, 0xC8.toByte, 0x32.toByte, 0x56.toByte, 0x35.toByte, 0xE3.toByte, 0x03.toByte)
```